src/ValueObject/SapiVersion.php

```diff
@@ -7,6 +7,9 @@
 use ValueObjects\Enum\Enum;
 
 /**
+ * @deprecated
+ *   Everything should use V3 by now so remove usages and this class.
```

Comment:
Sounds like a good candidate for a separate ticket?

Reply:
Created one: https://jira.uitdatabank.be/browse/III-4185